tests/driver_ble/Makefile

```diff
@@ -0,0 +1,17 @@
+APPLICATION = driver_ble
+include ../Makefile.tests_common
+
+BOARD = yunjia-nrf51822
+
+FEATURES_REQUIRED = nrf51_ble
+
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+USEMODULE += uart0
+USEMODULE += nrf51_ble
+USEMODULE += gnrc
+USEMODULE += gnrc_pktdump
```

Comment:
As far as I see it you don't use the `pktdump` module so you can take it out and you will save a whole thread! If you do so, you should be aware of some things. 1. See if you need to include another module which has been included through `pktdump` . 2. The interface you are interacting with in you your shell example will change (it is interface `3` then). You'll need to adapt the documentation then